colrev/ops/built_in/search_sources/eric.py

```diff
@@ -117,3 +280,7 @@ def prepare(
         """Source-specific preparation for ERIC"""
 
         return record
+
+
+if __name__ == "__main__":
```

Comment:
The `if __name__ == ...` can be removed because the Python file will always be called through the package (never as a standalone script).